CODING_GUIDELINES.md

```diff
@@ -370,9 +370,9 @@ Example:
 #include "ui_examplewidget.h"
 ```
 
-## 8. Include guard
+## 8. Include guards
```

Comment:
I would still keep Singular form (here and below). Otherwise, it looks as if we are talking about several types of "include guard". You don't need to use the Plural to indicate that "include guard" is used in multiple files, this is clear enough.

Reply:
@glassez OK, I was unsure about the best thing to do as well; your reasoning seems good to me, thanks for the clarification.